src/ngraph/frontend/onnx_import/onnx.hpp

```diff
@@ -40,6 +42,16 @@ namespace ngraph
                                const std::string& domain,
                                Operator fn);
 
+        /// \brief      Return the set of names of supported operators.
+        ///
+        /// \param[in]  version  The requested version of ONNX operators set.
+        /// \param[in]  domain   The requested domain the operators where registered for.
+        ///
+        /// \return     The set containing names of supported operators.
+        ///
+        std::set<std::string> get_supported_operators(std::int64_t version,
```

Comment:
We could add `get_supported_operators(std::int64_t version)` which assumes `ai.onnx` as the default domain.

Reply:
This can be achieved by the default parameter `get_supported_operators(std::int64_t version, std::string domain = "")` and then the user may just call `get_supported_operators(version)`.
However, I see no use case for such a scenario. By ONNX specification an operator name combines `domain` name and `version`.